ml-agents/mlagents/trainers/stats.py

```diff
@@ -234,14 +234,14 @@ def _maybe_create_summary_writer(self, category: str) -> None:
     def _delete_all_events_files(self, directory_name: str) -> None:
         for file_name in os.listdir(directory_name):
             if file_name.startswith("events.out"):
-                logger.warning(
+                logger.debug(
                     f"{file_name} was left over from a previous run. Deleting."
                 )
                 full_fname = os.path.join(directory_name, file_name)
                 try:
                     os.remove(full_fname)
                 except OSError:
-                    logger.warning(
+                    logger.debug(
```

Comment:
maybe `logger.error`? This one shouldn't happen unless we try to delete a file that has some write permissions blocked